cllc-public-app/ClientApp/src/app/components/applications/tied-house-decleration/tied-house-decleration-form/tied-house-declaration-form.component.html

```diff
@@ -17,10 +17,16 @@
         [valid]="isValidOrNotTouched('dateOfBirth')"
         errorMessage="Date of Birth is required">
         <input
-          class="form-control w-100"
-          type="date"
-          style="width: 100%; max-width: none"
-          formControlName="dateOfBirth" />
+          type="text"
```

Comment:
The field no longer shows as greyed out when its un-editable (in this case, editing an existing TH record).

<img width="884" height="529" alt="image" src="https://github.com/user-attachments/assets/80f0c131-d245-47e7-86ac-a672c8d1c65b" />